hackathon_site/dashboard/frontend/src/slices/hardware/hardwareSlice.test.ts

```diff
@@ -103,3 +106,67 @@ describe("getHardwareWithFilters thunk", () => {
         );
     });
 });
+
+describe("getHardwareNextPage thunk", () => {
+    it("Updates the store on API success", async () => {
+        const limit = mockHardware.length - 2;
+        const nextURL = `http://localhost:8000/api/hardware/hardware/?offset=${limit}`;
+        const { path, filters } = stripHostnameReturnFilters(nextURL);
+
+        const hardwareWithFiltersResponse = makeMockApiListResponse(
```

Comment:
idk how to pre-populate the store so I mocked getHardwareWithFilters for these 2 new tests

Reply:
You can use `const store = makeStoreWithEntities({ hardware: yourHardwareArray })`

Reply:
Nevermind, since that doesn't actually populate the next URL you can stick with the mocked responses!